src/site/content/en/blog/vitals-tools/index.md

```diff
@@ -102,12 +123,14 @@ Search Console is well-suited for both developers and those in non-developer rol
 
 Search Console may not be a fit for projects that use different third-party tools which group pages by similarity, or if a website isn't represented in the CrUX dataset.
 
+Page grouping can also be somewhat confusing when the example pages in that group have different characteristics of the group. For example, if the group fails particular Core Web Vitals overall, but the example pages all seem to pass the same Core Web Vitals. This can happen when a group contains a long tail or little visited pages that may be slower to load as less likely to be cached. When there are sufficient volumes of these pages in the long tail, they can influence the group's overall pass rate.
+
 ### Lighthouse
 
-[Lighthouse](https://developer.chrome.com/docs/lighthouse/overview/) is a lab tool that provides specific opportunities for improving page performance. [Lighthouse user flows](/lighthouse-user-flows/) also allow developers to script interaction flows for performance testing beyond page startup.
+[Lighthouse](https://developer.chrome.com/docs/lighthouse/overview/) is a lab tool that provides specific opportunities for improving page performance. [Lighthouse user flows](/lighthouse-user-flows/) also allow developers to script interaction flows for performance testing beyond page load.
 
 {% Aside %}
-Because Lighthouse is a lab tool and [First Input Delay (FID)](/fid/) is a field metric, it reports [Total Blocking Time](/tbt/) instead.
+Because Lighthouse is a lab tool and [First Input Delay (FID)](/fid/) and [Interaction to Next Paint (INP)](/inp/) are field metrics, it reports [Total Blocking Time](/tbt/) instead.
```

Comment:
```suggestion
Because Lighthouse is a lab tool, and [First Input Delay (FID)](/fid/) and [Interaction to Next Paint (INP)](/inp/) are field metrics, it reports [Total Blocking Time](/tbt/) instead.
```